runtime/service/src/main/java/org/apache/polaris/service/catalog/iceberg/IcebergCatalog.java

```diff
@@ -1566,14 +1566,44 @@ public void doCommit(TableMetadata base, TableMetadata metadata) {
               "Generic table with same name already exists: %s", tableIdentifier);
         }
       }
+      Map<String, String> storedProperties = new HashMap<>();
```

Comment:
Nit: possibly more readable when the construction of `storedProperties` is moved to a separate `static` function in `IcebergCatalog`.